Makefile

```diff
@@ -1,5 +1,6 @@
 .PHONY: help tests
 
+SHELL := /bin/bash
```

Comment:
Be careful here. This is being set globally for all OSes and Bash in Mac is __ancient__ by default. Some scripts may not work as intended.

Reply:
The proper fix, in any case, is to not use `#!/bin/sh` in our scripts but use `#!/bin/bash` instead. Or, avoid bashisms where not needed.

Reply:
Yeah, probably the best solution would be to refactor our scripts avoiding bash syntax. But I don't see that as a viable solution given the number of scripts we currently have. I' ll mark this conversation resolved as setting #!/bin/bash would have the same result.